README.md

```diff
@@ -211,6 +214,19 @@ The `subnav` field should be used to link to sections _within_ the current
 page, because links to other pages will cause the linking page's side
 navigation to collapse when visited.
 
+
+`sidenav` is a key _into_ `_data/navigation.yml`. See the [navigation](#navigation) docs for more info.
+
+A page's "current" or "active" state in the sidenav is
+determined by whether a link's `href` matches `page.url` or
+`page.permalink` for each page being rendered.
+
+`subnav` is a list of links to display on this page under its own link in the side navigation.
+
+**Note that subnav link hrefs are not prefixed with
+`site.baseurl`** because this breaks hash links prefixed with
+`#`.
```

Comment:
Is this just a documentation update? None of the `sidenav` functionality is changed in this pull request, right?

Reply:
That is correct